tt_metal/distributed/CMakeLists.txt

```
@@ -23,4 +25,6 @@ target_link_libraries(
         Metalium::Metal::Impl
         Metalium::Metal::LLRT
         TT::Metalium::HostDevCommon
+        ${MPI_CXX_LIBRARIES}
 )
+target_include_directories(distributed PRIVATE ${MPI_INCLUDE_PATH})
```

The docs ( https://cmake.org/cmake/help/latest/module/FindMPI.html ) say it declares an import target for us.  That'll handle both the lib + the incdir.
> MPI::MPI_\<lang>
Target for using MPI from \<lang>.

```suggestion
        MPI::MPI_CXX
)
```